### DISCUSSION TITLE
1.6.18: Docker: wo liegt die Feiertagsdatei?

### DISCUSSION THREAD

**Discussion by rolsch** [camueller/SmartApplianceEnabler#176] [Installation und Betrieb]
Hallo Axel,
ich kann aktuell die Holiday*.txt Datei nicht finden.

Im Volume sae finde ich nur die beiden *.xml der Config.
Wann wird diese Datei generiert?

Ps.: wie kontrolliere ich ob das SHM Broadcasting im Container ankommt?

Gruß
Roland

**Comment by camueller** [comment]
Die Holiday-Datei wird automatisch heruntergeladen, wenn das in den Einstellungen aktiviert ist.

Zur Kontrolle des SHM-Broadcasting wird es für SAE 2.x eine Anzeige im neuen SAE-Node-RED-Dashboard geben, die noch nicht implementiert ist (siehe https://github.com/camueller/SmartApplianceEnabler/issues/159).

**Comment by rolsch** [comment]
Hi Thomas.
Die Holiday*.txt wurde leider bisher nicht in meiner Dockerinstallation in der entsprechende HOST-Verzeichnis geschrieben.
- die beiden Config-XML jedoch schon...

docker-compose.yaml
...
    volumes:
      - './saedata/opt/sae/data:/opt/sae/data'
      #- './saedata/etc/localtime:/etc/localtime'
      #- './saedata/etc/timezone:/etc/timezone'
...

Ps.: Bezüglich Broadcasting dachte ich eher an entsprechende consolen cmd's aus dem Docker-Container heraus...

**Comment by camueller** [comment]
Wenn Anrede, dann bitte mit richtigem Namen. Zumal wir im PV-Forum schon einige Post ausgetauscht haben. Mein Name ist Axel.

Zu Deinem Docker-Setup kann ich nichts sagen. Auf meinem Docker-Raspi sind die Feiertage nicht aktiv. Ich prüfe das mal ...

Zum Thema "Broadcasting": In den Logs kannst Du erkennen, dass der SHM mit dem SAE kommuniziert. Die Logs sind ja auch im Docker-Container verfügbar.

**Comment by rolsch** [comment]
Huch, sorry, sowas aber auch :-)

Frag mich bitte nicht wie ich auf den falschen Namen komme...
Kommt davon wenn man mehrere Tabs mit Diskussionen gleichzeitig offen hat.

Danke für's prüfen und zum Broadcasting meinte ich so etwas wie wireshark oder iptraf-ng im Container ;-)

**Comment by camueller** [comment]
Ich habe gerade mal auf meinem Docker-Raspi die Feiertage aktiviert. Der Download klapper, aber erfolgt erst wenn auch ein Zeitplan vorhanden ist, für den Feitertage relevant sind.
